sources/Renderer/OpenGL/Platform/Android/AndroidGLSwapChainContext.cpp

```diff
@@ -63,7 +62,7 @@ void AndroidGLSwapChainContext::CanvasEventListener::OnInit(Canvas& sender)
 {
     /* Re-initialize the shared EGLSurface when the ANativeWindow is re-initialized */
     context_->InitEGLSurface(sender);
-    GLSwapChainContext::MakeCurrent(context_);
+    GLSwapChainContext::MakeCurrent(reinterpret_cast<GLSwapChainContext*>(context_));
```

Comment:
Do the Android source files need any changes? Might also be a left over from your original refactoring of the interface.